cypress/utils/utils.ts

```diff
@@ -1726,3 +1728,32 @@ export function performWithin(applicationName: string, actionFunction: () => voi
             actionFunction();
         });
 }
+
+/**
+ * Assigns credential to the list of applications
+ * @param appList is a list of applications where credential will be assigned
+ * @param credential is a credential to assign to those applications
+ */
+export function manageCredentialsForMultipleApplications(
+    appList: Application[],
+    credential: Credentials
+): void {
+    let selector: string;
+    Application.open();
+    appList.forEach((currentApp: Application) => {
+        currentApp.selectApplication();
+    });
+    clickWithin("#toolbar-kebab", button, false, true);
+    clickByText(button, manageCredentials);
+    // TODO: Add validation of application list, should be separated with coma
```

Comment:
**question:** I don't understand this: `should be separated with coma`

Reply:
it means, that if I do a bult credentials management, list of applications I am assigning creds to should look like `app1, app2, app3` in the management's modal window

Reply:
Refrased, should be more clear now

Reply:
gotcha! thanks for clarifying!